pol-core/pol/polvar.h

```diff
@@ -13,7 +13,9 @@ namespace Core
 class PolVar
 {
 public:
-  unsigned short DataWrittenBy;
+  std::string DataWrittenBy;
```

Comment:
In theory this class needs also a estimateSize method, since the string needs a capacity call. The string is not really long so it will not make a big difference.

Reply:
Thanks!  Added for PolVar, though there are other members of SettingsManager (Party_Cfg, ServSpecOpt) that are not included in its estimate.